source/jormungandr/tests/authentication_tests.py

```diff
@@ -27,17 +27,20 @@
 # https://groups.google.com/d/forum/navitia
 # www.navitia.io
 from __future__ import absolute_import, print_function, unicode_literals, division
-import logging
 from jormungandr.tests.utils_test import MockResponse, user_set, FakeUser
 from navitiacommon import models
 
 from .tests_mechanism import AbstractTestFixture, dataset
 from .check_utils import *
 from jormungandr import app
 import json
-from nose.util import *
 import mock
 
+"""
+coords in main_routing_test coverage
+"""
+coords = '{lon};{lat}'.format(lon=0.000628818387368, lat=0.00107797437835)
```

Comment:
can't you use https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/tests/check_utils.py#L1124 ?